### PR TITLE
search city field

### PR DESCRIPTION
**Pull request by c3soft**
removal of component Header from the home page and
creation of a search field by city

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Node Modules here

**Comment by c3soft** [comment]
delete node module to root project

File: opus/src/App.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import './App.css';
 
-import Header from './components/header/Header';
+//import Header from './components/header/Header';
```
Comment (ghost) [inline opus/src/App.js]: Please remove dead code

File: opus/src/App.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ class App extends React.Component {
   render() {
     return (
       <div className="App">
-        <Header />
+        {/* <Header /> */}
```
Comment (ghost) [inline opus/src/App.js]: Please remove dead code

File: opus/src/components/connection/conn.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React, { Component } from 'react';
+
+export default class conn extends Component {
```
Comment (ghost) [inline opus/src/components/connection/conn.js:3]: Even if this component isn't used yet, you can still keep conventions (explicit name & PascalCase)

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Plus new conflicts on App.js

**Comment by c3soft** [comment]
delete dead code

**Comment by ghost** [comment]
Still conflict on App

**Pull request merged**